src/iop/filmicrgb.c

```diff
@@ -114,9 +114,8 @@ typedef enum dt_iop_filmicrgb_methods_type_t
   DT_FILMIC_METHOD_MAX_RGB = 1,           // $DESCRIPTION: "max RGB"
   DT_FILMIC_METHOD_LUMINANCE = 2,         // $DESCRIPTION: "luminance Y"
   DT_FILMIC_METHOD_POWER_NORM = 3,        // $DESCRIPTION: "RGB power norm"
-  DT_FILMIC_METHOD_EUCLIDEAN_NORM_V2 = 5, // $DESCRIPTION: "RGB euclidean norm"
   DT_FILMIC_METHOD_EUCLIDEAN_NORM_V1 = 4, // $DESCRIPTION: "RGB euclidean norm (legacy)"
-  DT_FILMIC_METHOD_LAST = 6, // this must always be greatest of the numbers above + 1
+  DT_FILMIC_METHOD_EUCLIDEAN_NORM_V2 = 5, // $DESCRIPTION: "RGB euclidean norm"
```

Comment:
The practice is (seems to be...) to explicitly number all enums that are part of params for backward compatibility.

By putting DT_FILMIC_METHOD_EUCLIDEAN_NORM_V1 in the position corresponding to its number, it is possible to directly use that in `dt_bauhaus_combobox_remove_at`. Otherwise either we must assume it is always at the end (and we could use _LAST-1), or we need a new function `dt_bauhaus_combobox_remove_value`. I didn't think it was worth introducing that for the _very_ few cases that would use (and _test_) it.

I think this kind of stuff will always be somewhat messy and needs people to know exactly what they are doing, so I considered it not worth trying to make it "fool" proof (and encouraging people to use it thoughtlessly).

Reply:
That's good, thanks for the explanation.